<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Counter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_Plan">
      <item row="0" column="0">
       <widget class="QLabel" name="label_Plan">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enter data plan</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_dataBytes">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>MB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GB</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="spinBox_dataValue">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
        <property name="value">
         <number>1024</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout_Results">
      <item row="0" column="0">
       <widget class="QLabel" name="label_Results">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Results</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_Calculate">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_fullNumber">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Show full number</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEdit_result">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>47</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>2147483647</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Close</set>
      </property>
      <property name="centerButtons">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="mainLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mobile data estimate counter</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="closeCountedSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_monthsDays">
      <item row="2" column="0">
       <widget class="QLabel" name="label_Month">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Select month</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="comboBox_Months">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>January</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>February</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>March</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>April</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>May</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>June</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>July</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>August</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>September</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>October</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>November</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>December</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_Days">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-Days</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="spinBox_Days">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QPushButton" name="pushButton_getDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Get current date</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_customDays">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Use custom amount of days</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
